cobc/codegen.c

```diff
@@ -1830,16 +1830,28 @@ output_gnucobol_defines (const char *formatted_date)
 	output_line ("#define  COB_MODULE_TIME\t\t%d", i);
 
 	{
-		struct cb_text_list *l = cb_include_file_list ;
-		for (;l;l=l->next){
-			if (l->text[0] == '<'){
+		struct cb_text_list *l;
+		for (l = cb_include_file_list; l; l = l->next) {
+			if (l->text[0] == '<') {
 				output_line ("#include %s", l->text);
 			} else {
 				output_line ("#include \"%s\"", l->text);
 			}
 		}
-	}
 
+		l = cb_include_file_list_directive;
+		while (l) {
```

Comment:
What is the reason for using a "full for loop" above (that's fine) and here go with a different one?`If there isn't any please use `for` in both cases.